REFERENCE.md

```diff
@@ -0,0 +1,418 @@
+<p align="center">
+  <a href="https://www.airflow.apache.org">
+    <img alt="Airflow" src="https://cwiki.apache.org/confluence/download/attachments/145723561/airflow_transparent.png?api=v2" width="60" />
+  </a>
+</p>
+<h1 align="center">
+  Astro SDK Python :rocket:
+</h1>
+  <h3 align="center">
+  Your new Airflow DAG writing experience. Maintained with ❤️ by Astronomer.
+</h3>
+<br/>
+
+# :mega: Disclaimer :mega:
+```
+This project is in a previous release. Interfaces may change with feedback from early users.
```

Comment:
```suggestion
This project is in a **preview** release. Interfaces may change with feedback from early users.
```

Something about "This project is in a **preview** release" still doesn't sound grammatically correct